app/controllers/prisons_application_controller.rb

```diff
@@ -23,4 +33,9 @@ def check_prison_access
 
     @prison = Prison.new(active_prison_id)
   end
+
+  def pom_at_active_prison?
+    user = Nomis::Elite2::UserApi.user_details(current_user)
+    StaffMember.new(user.staff_id).pom_at?(active_prison_id)
```

Comment:
This feels like it should be a member variable like @prison which is available to all controllers via inheritance. Then we load it in a before action, and pom_at_active_prison? becomes @current_staff_member.pom_at?(active_prison_id)